api/src/test/java/org/openmrs/module/pihmalawi/reporting/reports/IC3MonthlyIndicatorReportTest.java

```diff
@@ -52,7 +52,7 @@ public void performTest() throws Exception {
         String user = "root";
         String password = "root";
 
-        File propertiesFile = new File(OpenmrsConstants.APPLICATION_DATA_DIRECTORY, PihMalawiConstants.OPENMRS_WAREHOUSE_CONNECTION_PROPERTIES_FILE_NAME);
+        File propertiesFile = new File(OpenmrsConstants.APPLICATION_DATA_DIRECTORY_RUNTIME_PROPERTY, PihMalawiConstants.OPENMRS_WAREHOUSE_CONNECTION_PROPERTIES_FILE_NAME);
```

Comment:
This is not correct.  This constant value is "application_data_directory" and not the actual application data directory that is set.  You should use OpenmrsUtil.getApplicationDataDirectory() to get the base directory here.

Reply:
Thanks @mseaton ! I have updated all those APPLICATION_DATA_DIRECTORY lines.